common/djangoapps/static_replace/__init__.py

```diff
@@ -180,7 +180,7 @@ def replace_static_url(original, prefix, quote, rest):
             else:
                 # if not, then assume it's courseware specific content and then look in the
                 # Mongo-backed database
-                url = StaticContent.convert_legacy_static_url_with_course_id(rest, course_id)
+                url = StaticContent.get_canonicalized_asset_path(course_id, rest)
```

Comment:
I mean, it's possible, but then every other usage we end up with for `get_canonicalized_asset_path` will need to also do the same thing.  I guess it just feels wrong that you're saying "hey, can you please canonicalize this asset path for me? I'm not sure what it should look like, but I know for sure this is the domain you should serve it from."

I definitely prefer a configuration value that is configurable at runtime vs needing a deploy, it's just... bah. :P